src/test/java/Calculator/CalculatorTest.java

```diff
@@ -0,0 +1,102 @@
+package Calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculatorTest {
+
+    @Test
+    @DisplayName("계산기 전체테스트")
+    void allTest() {
+        //given
+        Calculator calculator = new Calculator("2 + 3 * 4 / 2");
+
+        //when
+        int actual = calculator.calculate();
+
+        //then
+        int expected = 10;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("덧셈")
+    void plus() {
+        //given //when
+        Calculator calculator = new Calculator();
+        int actual = calculator.add(3, 5);
+
+        //then
+        int expected = 8;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("뺄셈")
+    void minus() {
+        //given //when
+        Calculator calculator = new Calculator();
+        int actual = calculator.minus(3, 5);
+
+        //then
+        int expected = -2;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("곱셈")
+    void multiply() {
+        //given //when
+        Calculator calculator = new Calculator();
+        int actual = calculator.multiply(3, 5);
+
+        //then
+        int expected = 15;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("나눗셈")
+    void divide() {
+        //given //when
+        Calculator calculator = new Calculator();
+        int actual = calculator.divide(4, 2);
+
+        //then
+        int expected = 2;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("입력 값이 null이거나 빈 공백 문자일 경우 IllegalArgumentException throw")
+    void inputIsNulloreEmpty() {
+        //given
+        String inputString = "";
+
+        //when
+        Exception actual = assertThrows(IllegalArgumentException.class, () -> {
+            Calculator calculator = new Calculator(inputString);
+        });
```

Comment:
`입력 값이 null이거나 빈 공백 문자일 경우..`를 테스트한다고 작성하신 것 같은데
실제 테스트 로직에는 빈 문자열만 들어가고 있습니다.

이런 상황에서는 JUnit의 어노테이션을 적절하게 사용하면 해결할 수 있어요 😄
```suggestion
    @ParameterizedTest(name = "\"{0}\" 입력 시 IllegalArgumentException throw")
    @NullAndEmptySource
    void inputIsNulloreEmpty(String inputString) {
        //when
        Exception actual = assertThrows(IllegalArgumentException.class, () -> {
            Calculator calculator = new Calculator(inputString);
        });
```